### ISSUE TITLE
[Rich text editor] CTRL + Backspace delete the whole text instead the last word

### ISSUE THREAD

**Issue by JokerGermany** [vector-im/element-web#25645]
#### What would you like to do?
Delete the word the next word which is next to the cursor

#### How would you like to achieve it?
Pressing CTRL+Backspace
(Like with the "old" Editor)

### Additional context

CTRL+Backspace delete the whole text.
I don't know any application which do it that way. They only delete one word.

It's really hard for me to use the Rich Text Editor, because it's a habit using CTRL+Backspace to delete the last word.

(Don't know if i should classify it as a bug or feature...)

**Comment by frebib** [comment]
I don't think this is a problem any more, but instead it's replaced by a new bug -> #26647
